pkgs/development/libraries/gdk-pixbuf/default.nix

```diff
@@ -58,7 +62,18 @@ stdenv.mkDerivation rec {
     libxslt
     docbook-xsl-nons
     docbook_xml_dtd_43
-  ] ++ lib.optional stdenv.isDarwin fixDarwinDylibNames;
+  ] ++ lib.optionals stdenv.isDarwin [
+    fixDarwinDylibNames
+  ] ++ lib.optionals withIntrospection [
+    gobject-introspection
+  ];
+
+  buildInputs = [
+    python3
+    gi-docgen
```

Comment:
Why is this in `buildInputs`? Same with g-i.

Reply:
found out that only pkg-config has to be in depsBuildBuild and "it just works"